src/python/visclaw/data.py

```diff
@@ -25,7 +25,7 @@ class ClawPlotData(clawdata.ClawData):
     """
 
     # ========== Initialization routine ======================================
-    def __init__(self, controller=None):
+    def __init__(self, controller=None, num_dim=2):
```

Comment:
True but for anything that uses `num_dim` and is not 2D this will break.  Since nothing really uses `num_dim` maybe this won't be as bad as I was first suggesting but the parameter will be set incorrectly everywhere but for 2D runs.